packages/template-retail-react-app/app/pages/account/wishlist/partials/wishlist-secondary-button-group.jsx

```diff
@@ -36,10 +36,18 @@ export const REMOVE_WISHLIST_ITEM_CONFIRMATION_DIALOG_CONFIG = {
         defaultMessage: 'Yes, remove item',
         id: 'confirmation_modal.remove_wishlist_item.action.yes'
     }),
+    primaryActionAriaLabel: defineMessage({
+        defaultMessage: 'Yes, remove item',
```

Comment:
This is for a11y. For some modals, they are all the same, but for other like the unavailableProductModals, it is differently to add more context on the button. The UI shows "Remove" but aria-text is "Remove unavailable products" Also, this is helpful to get the element in unit test since the ariaLabel is more unit than the UI text.